Sources/Nuke/Decoding/ImageDecoders+Default.swift

```diff
@@ -52,7 +52,9 @@ extension ImageDecoders {
 
             func makeImage() -> PlatformImage? {
                 if let thumbnail {
-                    return makeThumbnail(data: data, options: thumbnail)
+                    return makeThumbnail(data: data, 
```

Comment:
use scale in creating a thumbnail similar to how `ImageDecoders.Default._decode(data, scale: scale)` is using it below